packages/quick_actions/quick_actions_ios/example/integration_test/quick_actions_test.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.9
```

Comment:
Do we still need this? I thought they had all been removed from the repo; it looks like this in one of a couple that we somehow missed.

Reply:
It could indeed be removed by specifying a dummy callback parameter instead of `null` to the `QuickActions.initialize()` method.